Bottom: bcac157fe4319f286d40e44f6f1414ae79261a58
Top:    50ab41d185d366054aa5519e3090748bd9eda7a4
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 18:09:44 -0700

Refresh of iom-dbg

---

diff --git a/src/rsocket.c b/src/rsocket.c
index a26136d..09933f5 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -2331,7 +2331,11 @@ off_t riomap(int socket, void *buf, size_t len, int prot, int flags, off_t offse
 	printf("%s\n", __func__);
 	rs = idm_at(&idm, socket);
 	if ((rs->state != rs_connect_rdwr) || (prot & ~(PROT_WRITE | PROT_NONE)))
+	{
+		printf("%s - invalid state ((%x) %d prot %d \n", __func__,
+			rs->state, (rs->state != rs_connect_rdwr), (prot & ~(PROT_WRITE | PROT_NONE)));
 		return ERR(EINVAL);
+	}
 
 	fastlock_acquire(&rs->iomap_lock);
 	if (prot & PROT_WRITE) {
